.github/buildomat/jobs/host-image.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 #:
-#: name = "helios / build OS image"
+#: name = "helios / build OS images"
 #: variety = "basic"
 #: target = "helios-2.0"
 #: rust_toolchain = "1.72.1"
 #: output_rules = [
-#:	"=/work/helios/image/output/os.tar.gz",
+#:	"=/work/helios/upload/os-host.tar.gz",
+#:	"=/work/helios/upload/os-trampoline.tar.gz",
```

Comment:
Seeing as we're moving them around anyway, we could just as well:

```
mkdir -p /helios
```

and put them in there.  They don't need to be under `/work` etc.

Reply:
It doesn't hurt to put them under `/work` though, I think. Then there is consistency in the tuf-repo script with the other things it's picking up from jobs.

It's a read-only root, so `/helios` doesn't work.

Reply:
It's definitely not a read-only root.  It's just a regular VM with a mutable disk installation; that's why you can use `pkg install` etc.  e.g., see https://github.com/oxidecomputer/pilot/blob/main/.github/buildomat/jobs/bins.sh

Reply:
My mistake,  `pfexec` was what I needed there.